crates/telio-network-monitors/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![deny(missing_docs)]
+
+//! Network monitor module
+pub mod monitor;
+
+#[cfg(any(target_os = "macos", target_os = "ios", target_os = "tvos"))]
+/// Utility to get monitor network on apple
+pub mod mac;
```

Comment:
IMHO, the name for this mod is inaccurate, "mac" doesn't describe this group of OSes.. eg: "darwin" would be more suitable.